pkg/v2/relay/cmd/server/main.go

```diff
@@ -24,6 +26,9 @@ func main() {
 
 	var port uint = 0
 	flag.UintVar(&port, "port", 0, "the port to run on for the websocket")
+	var staticMetricsFilename string
```

Comment:
2 things
1. move the code to a package pkg/v2/metrics
2. make it so that persistence is configurable (via some sort of interface)

the reason being is that i am going to want to create a persistence that sends down stats to my local machine every ~10 seconds or so

please respond to this right away if you work on it.  I am going to take care of it at some point and if you do it before me, awesome, if not, i'll do it :)

Reply:
> please respond to this right away if you work on it. I am going to take care of it at some point and if you do it before me, awesome, if not, i'll do it :)

I've got a little time now so gonna have a crack at it.